server/index.js

```diff
@@ -2,16 +2,25 @@ import express from 'express';
 import dotenv from 'dotenv';
 
 import dbConfig from './config/db';
+import middlewaresConfig from './config/middlewares';
+import { MeetupRoutes } from './modules';
 
 dotenv.config();
 
 const app = express();
 
 /*
-* Database
+*   Database
 */
 dbConfig();
 
+/*
+*   Middlewares
+*/
+middlewaresConfig(app);
+
+app.use('/api', [MeetupRoutes]);
+
 const PORT = process.env.PORT || 3000;
 
 app.listen(PORT, err => {
```

Comment:
I personally prefer to use brackets always: 
`app.listen(PORT, (err) => {`

Reply:
Oh good catch there.